### DISCUSSION TITLE
Example generator without input/output

### DISCUSSION THREAD

**Discussion by NeoMystical** [NCBPFluffyBear/SlimeCustomizer#6] [Q&A]
Hello
Can you create Example code for Generator without input source like Solar panel

**Comment by NCBPFluffyBear** [comment]
Solar panels are currently not a thing, but it is a planned feature

**Reply by NCBPFluffyBear** [reply]
![image](https://user-images.githubusercontent.com/31554056/115166699-24df4b80-a07a-11eb-822b-34a00c337e2f.png)
